servant-js/test/Servant/JSSpec.hs

```diff
@@ -81,12 +85,16 @@ spec = describe "Servant.JQuery" $ do
     axiosSpec
     --angularSpec    AngularCustom
 
+
```

Comment:
Yes will add them.   They're not exported though as they're just internal helper functions.  But I guess it's prettier to add a signature to them